barcodes_generator_lot/README.rst

```diff
@@ -0,0 +1,103 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Generate Barcodes for Stock Production Lots
+===========================================
+
+This module expands Odoo functionality, allowing user to generate barcode
+depending on a given barcode rule for Stock Production Lots.
+
+The barcode will be used as the ``name`` of the lot, which effectively allows
+this module to create automatic serialization rules for production lots.
+
+For example, a typical pattern for partners is  "042........." that means
```

Comment:
Are there some nomenclature for lots ? If, yes maybe you could change the description of the module. (partner sample is not very relevant for that module).
(just a question, I don't use lot)